File: FinancialMonitoring/ClientApp/src/app/models/finance/occurenceKind.ts
/**
 * The occurence kind for a finacial item
 */
export enum OccurenceKind
{
  /**
   * The FinancialItem occures only one time 
   */
  OneTime = "OneTime",
  /**
   * The FinancialItem occures every month
   */
  Monthly = "Monthly",
  /**
   * The FinancialItem occures every year
   */
  Yearly = "Yearly"
}

File: FinancialMonitoring.Entities/Finance/OccurenceKind.cs
namespace FinancialMonitoring.Entities.Finance
{
  /// <summary>
  /// The occurence kind for a finacial item
  /// </summary>
  public enum OccurenceKind
  {
    /// <summary>
    /// The FinancialItem occures only one time 
    /// </summary>
    OneTime,
    /// <summary>
    /// The FinancialItem occures every month
    /// </summary>
    Monthly,
    /// <summary>
    /// The FinancialItem occures every year
    /// </summary>
    Yearly
  }
}